src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main">
+      <div class="calendar calendar--start-day_wen calendar--month-length-29">
+        <div class="calendar__cell cell-1"></div>
```

Comment:
```suggestion
        <div class="calendar__cell cell-1"></div>
```
cell-1 модификатор по бему? или элемент? Не используй модификаторы для каждой клетки, достаточно одного класса calendar__cell

Reply:
Да, это модификатор. Сейчас перепишу. Я так понимаю, что должно быть <div class="calendar__cell calendar__cell--1"></div>
cell-1 и тд использованы только для того что бы проставить им числа при помощи ::before.
Как это сделать не нумеруя каждую клетку - ума не приложу.